bin/add-font-assets.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+const fs = require('fs');
+
+const json = fs.readFileSync('package.json', 'utf8');
+
+const pack = JSON.parse(json);
+
+if (!pack.rnpm) {
+  pack.rnpm = {
+    assets: [],
+  };
+} else if (!pack.rnpm.assets) {
+  pack.rnpm.assets = [];
+}
+
+for (let i = 0; i < pack.rnpm.assets.length; i += 1)
+  if (pack.rnpm.assets[i] === './assets/fonts') process.exit();
```

Comment:
Feels like `includes()` or an old school `pack.rnpm.assets.indexOf('./assets/fonts') !== -1` would be prettier somehow

Reply:
Total blackout here, of course that's what it should be!